### PR TITLE
Fix replay mode

### PR DESCRIPTION
**Pull request by aby2s**

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = tests
+#addopts = tests
```
Comment (ksaaskil) [inline pytest.ini]: Are we getting rid of this? If yes, I'd change the documentation and possibly setup.py test command to include "tests", perhaps.

File: tests/build/test_builder.py [hunk 2]
```diff
@@ -112,7 +112,19 @@ def test_pokeapi_schema_valid(schema):
     assert_that(paths, has_item(
         matches_regexp(r'\/v2\/ability\/\{[\w]+\}')))
 
-#### petstore
+
+def test_pokeapi_schema_valid_replay(schema):
+    pokeapi_schema = build_schema_batch(pokeapi_requests, UpdateMode.REPLAY)
+    paths = list(pokeapi_schema.paths.keys())
+    assert 14 == len(paths)
```
Comment (ksaaskil) [inline tests/build/test_builder.py]: Maybe for a bit easier debugging use hamcrest?
```suggestion
    assert_that(paths, has_length(14))
```

File: meeshkan/build/media_types.py [hunk 3]
```diff
@@ -25,10 +25,9 @@ def update_text_schema(text_body: str, mode: UpdateMode, schema: Optional[Any] =
     # TODO Better updates
     generic = Schema(_type="string")
     specific = Schema(_type="string", enum=[text_body])
-    return generic if mode == UpdateMode.GEN else Schema(oneOf=list(set([
-            specific,
-            *([] if schema is None else [schema] if schema.oneOf is None else schema.oneOf)
-        ]))
+    # TODO don't merge equal schemas
+    return generic if mode == UpdateMode.GEN else Schema(
+        oneOf=[specific, *([] if schema is None else [schema] if schema.oneOf is None else schema.oneOf)]
```
Comment (ksaaskil) [inline meeshkan/build/media_types.py]: What does this fix?

Comment (aby2s) [reply meeshkan/build/media_types.py]: Any update mode besides GEN was broken after migration to dataclasses.

File: pytest.ini (hunk 1 above)
Comment (aby2s) [reply pytest.ini]: No, I committed it accidentally.

**Pull request merged**